storage/Cargo.toml

```diff
@@ -12,8 +15,8 @@ arc-swap = "=0.4"
 base64 = { version = "0.13.0", optional = true }
 bitflags = "1.2.1"
 futures = "0.3"
-governor = "0.4"
-hmac = { version = "0.8.1", optional = true }
+governor = "=0.4.1"
```

Comment:
Sure to use `=` ?

Reply:
Add comments for it
"# pin governor to avoid multi versions of hashbrown"